drivers/display/display_ili9340_seeed_tftv2.c

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2018 - 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <display.h>
+#include "display_ili9340.h"
+
+/*
+ * Derive from Seeed 2.8 inch TFT Touch Shield v2.0 sample code.
+ *
+ * https://github.com/Seeed-Studio/TFT_Touch_Shield_V2
+ */
+
+void ili9340_lcd_init(struct ili9340_data *p_ili9340)
+{
+	u8_t cmd;
+	u8_t data[15];
+
+	/* Software reset */
+	cmd = 0x01U;
+	ili9340_transmit(p_ili9340, cmd, NULL, 0);
+
+	k_sleep(5);
+
+	cmd = 0xCFU;
```

Comment:
@leewkb4567 @vanwinkeljan Are you happy with an other file full of magics? The command should be added to display_ili9340.h and readable defines used here. The same for data[.] magics, please add the register or register flags. This should be readable and understandable by humans.

Reply:
The Seeed LCD shield works when I apply their sample code to nRF52 DK. Actually I do not understand their command sequence either. The LCD shield vendors rarely offer any documentation to program their display.

Anyway, I will update this Seeed driver code to make it easier to read.

Reply:
The Seeed LCD initialization has been updated with ILI9341 command numbers.

Some command numbers are not found in ILI9341 datasheet V1.02. They are found in newer datasheet V1.11. A PDF copy can be located in following URL:

https://www.newhavendisplay.com/app_notes/ILI9341.pdf

Please review again. Thanks.